samples/charger_and_events/README.rst

```diff
@@ -0,0 +1,108 @@
+.. _npmx_charger_and_events_sample:
+
+npmx: CHARGER and EVENTS
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The npmx charger and events sample demonstrates how to control PMIC charger and receive events using npmx drivers.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires an nPM1300 EK.
+
+Overview
+********
+
+This sample uses changer and events drivers to do the following:
+
+* Set charger's basic configuration.
+* Register events from device.
+
+Wiring
+******
+
+#. Connect the TWI interface between the chosen DK and the nPM1300 EK as in the following table:
+
+   .. list-table:: nPM1300 EK connections.
+      :widths: 25 25 25
+      :header-rows: 1
+
+      * - nPM1300 EK pins
+        - nRF5340 DK pins
+        - nRF52840 DK pins
+      * - GPIO0
+        - P1.10
+        - P1.10
+      * - SDA
+        - P1.02
+        - P0.26
+      * - SCL
+        - P1.03
+        - P0.27
+      * - VOUT2 & GND
+        - External supply (P21)
+        - External supply (P21)
+
+#. Make the following connections on the nPM1300 EK:
+
+   * Connect a USB power supply to the **J3** connector.
+   * Connect a suitable battery to the **J2** connector.
+   * On the **P18** pin header, connect **VOUT2** and **VDDIO** pins with a jumper.
+   * On the **P2** pin header, connect **VBAT** and **VBATIN** pins with a jumper.
+   * On the **P17** pin header, connect all LEDs with jumpers.
+   * On the **P13** pin header, connect **RSET1** and **GND** pins with a jumper.
+   * On the **P14** pin header, connect **RSET2** and **VSET2** pins with a jumper.
+
+#. Make the following connections on the DK:
+
+   * Set the **SW9** nRF power source switch to **VDD**.
+   * Set the **SW10** VEXT → nRF switch to **ON**.
+
+Configuration
+*************
+
+|config|
+
+Configuration options
+=====================
+
+Check and configure the following sample-specific Kconfig options:
+
```

Comment:
What about `BATTERY_VOLTAGE_THRESHOLD_1` and `_2`?

Reply:
Added.